### ISSUE TITLE
Fix tests (richardstartin)

### ISSUE THREAD

**Issue by gunnarmorling** [gunnarmorling/1brc#103]
Hey @richardstartin, @AlexanderYastrebov added one more test for 10K distinct keys which fails with your impl. Could you take a look? Thx!

```
./test.sh richardstartin
Validating calculate_average_richardstartin.sh -- src/test/resources/samples/measurements-1.txt

real	0m0.077s
user	0m0.077s
sys	0m0.019s
Validating calculate_average_richardstartin.sh -- src/test/resources/samples/measurements-10.txt

real	0m0.065s
user	0m0.076s
sys	0m0.014s
Validating calculate_average_richardstartin.sh -- src/test/resources/samples/measurements-10000-unique-keys.txt
Exception in thread "main" java.lang.ArrayIndexOutOfBoundsException
	at java.base/jdk.internal.reflect.DirectConstructorHandleAccessor.newInstance(DirectConstructorHandleAccessor.java:62)
	at java.base/java.lang.reflect.Constructor.newInstanceWithCaller(Constructor.java:502)
	at java.base/java.lang.reflect.Constructor.newInstance(Constructor.java:486)
	at java.base/java.util.concurrent.ForkJoinTask.getThrowableException(ForkJoinTask.java:542)
	at java.base/java.util.concurrent.ForkJoinTask.reportException(ForkJoinTask.java:567)
	at java.base/java.util.concurrent.ForkJoinTask.join(ForkJoinTask.java:653)
	at dev.morling.onebrc.CalculateAverage_richardstartin.main(CalculateAverage_richardstartin.java:391)
Caused by: java.lang.ArrayIndexOutOfBoundsException: Index 603 out of bounds for length 600
	at dev.morling.onebrc.CalculateAverage_richardstartin$Page.update(CalculateAverage_richardstartin.java:254)
	at dev.morling.onebrc.CalculateAverage_richardstartin$AggregationTask.computeSlice(CalculateAverage_richardstartin.java:307)
	at dev.morling.onebrc.CalculateAverage_richardstartin$AggregationTask.compute(CalculateAverage_richardstartin.java:339)
	at dev.morling.onebrc.CalculateAverage_richardstartin$AggregationTask.compute(CalculateAverage_richardstartin.java:276)
	at java.base/java.util.concurrent.RecursiveTask.exec(RecursiveTask.java:110)
	at java.base/java.util.concurrent.ForkJoinTask.doExec(ForkJoinTask.java:387)
	at java.base/java.util.concurrent.ForkJoinPool$WorkQueue.topLevelExec(ForkJoinPool.java:1312)
	at java.base/java.util.concurrent.ForkJoinPool.scan(ForkJoinPool.java:1843)
	at java.base/java.util.concurrent.ForkJoinPool.runWorker(ForkJoinPool.java:1808)
	at java.base/java.util.concurrent.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:188)
```

**Comment by richardstartin** [comment]
Fixed in #108

**Comment by gunnarmorling** [comment]
Done.
